packages/lesswrong/server/manualMigrations/2023-07-28-viewUpdaterErrorHandling.ts

```diff
@@ -0,0 +1,73 @@
+import { getSqlClientOrThrow } from '../../lib/sql/sqlClient';
+import { registerMigration } from './migrationUtils';
+
+// This updates the functions created in packages/lesswrong/server/manualMigrations/2023-01-13-createViewUpdater.ts
+// to handle the case where a table name is already lowercase
+registerMigration({
+  name: "viewUpdaterErrorHandling",
+  dateWritten: "2023-07-28",
+  idempotent: true,
+  action: async () => {
+    const db = getSqlClientOrThrow();
+
+    const sql = `
+    create or replace function refresh_lowercase_views() returns void as
+    $$
+    DECLARE
+        rec record;
+    BEGIN
+      FOR rec IN
+        select table_name,
+          string_agg(format('%I as %s', column_name, column_name), ', ') cols
+        from information_schema."columns"
```

Comment:
```suggestion
        from information_schema.columns
```

I don't think you need any of these quotes in this function.

Reply:
I acknowledge that you haven't changed the original function here.

Reply:
I agree they're not necessary. I'd rather keep them though seeing as were doing a lot of case-fiddling here, I think it makes it less confusing to have everything in quotes by default so you can more easily work out in which parts the case is actually important